lambda_function.py

```diff
@@ -293,64 +189,200 @@ def gather_table_stats(cursor, cluster):
             {
                 'MetricName': 'Rows',
                 'Dimensions': [
-                    { 'Name': 'ClusterIdentifier', 'Value': cluster}
+                    {'Name': self.environment, 'Value': self.cluster}
                 ],
                 'Timestamp': datetime.datetime.utcnow(),
                 'Value': total_rows,
                 'Unit': 'Count'
             }
-        ] 
-    
-       
+        ]
+
+    def run_external_commands(self, command_set_type, file_name, cursor):
+        if not os.path.exists(file_name):
+            return []
```

Comment:
Would it be better to, in addition, at least log something here and on line 209 when we're returning empty lists? at least as debug?